eureka/tests/test_general.py

```diff
@@ -5,7 +5,10 @@
 
 import numpy as np
 from ..lib import util
-from ..S2_calibrations import s2_calibrate as s2
+try:
```

Comment:
I imagine anyone testing the pipeline will have the jwst packaged installed.

Reply:
Well I wanted to be able to test that the pipeline would work as intended without jwst installed too though, otherwise it was hard for me to tell what I might have broken